### ISSUE TITLE
Limit

### ISSUE THREAD

**Issue by fnxpt** [thomasloven/lovelace-auto-entities#41]
In some scenarios is also good to limit the amount of devices listed, is there a way to do this.

**Comment by thomasloven** [comment]
I might add it if you give a convincing use case and configuration syntax suggestion.

**Comment by fnxpt** [comment]
I used to use this with flex-table-card, but I prefer this card.
I used it mainly for Transport schedules, so I can keep track of the next bus/metro near my house.
I have around 12 sensors, but I only want to see 4 or 5...

**Comment by Fredderic** [comment]
> I might add it if you give a convincing use case and configuration syntax suggestion.

This would be awesome, in combination with the "most recently changed sort" issue.

Also…

I have a thing I created which does the "days until XXX" with some bonus extras; it handles both exact date and internet lookup dates, only updates those entities once a year, and lastly, creates a "next" entity that as the name implies, is basically a copy of the next event due (at present, that turns out to be the summer solstice).  An additional attribute on that "next" event, is a list of all the events in sorted order (since I needed to sort them, but didn't have a use for the list, I figured that was as good a place to stash it as any for the time being).
Now, I'm presently showing just the next entity in a badge, and was considering adding next2 and next3 entities, but wasn't real happy with that idea.  If I could instead wrangle that list into a group, an option like this would let me show, say, the next three events in a nice neat little glance card.
